dashboard/test_d_graphs.py

```diff
@@ -0,0 +1,188 @@
+# pylint skip-file
+"""Testing the d_graphs file"""
+import re
```

Comment:
Could also add a space here between native and non-native libraries, but small detail!

Reply:
Good catch - will make the changes